plays/plays.md

```diff
@@ -152,6 +152,7 @@ Once a set of solutions is prioritized, the state and vendor work together to sc
 
 #### Guides
 * [Benefits Data Trust: Prioritization Tool](https://bdtrust.org/Prioritization_Tool.docx): Determine what policy and process solutions are feasible by analyzing the risk, value, cost and effort of implementation and establish action steps for implementing proposed policy & process changes.
+* [Technical System Overivew](../resources/system-overview.md) provides a general overview of the system components you should expect to find supporting a Medicaid program, as well as some common shortfalls and limitations.
```

Comment:
Spelling: `Overview`

Reply:
🤦 I read the non-link text part like 5 times... done. thank you